jetson/percep_obs_detect/src/common.cu

```diff
@@ -0,0 +1,94 @@
+#include "common.hpp"
+#include <iostream>
+#include <sl/Camera.hpp>
+
+#include <vector>
+#include <algorithm>
+
+// TODO: add comments explaining the purpose of these functions
```

Comment:
As long as you have briefs for these in common.hpp, which I believe you do, you can remove this comment